scripts/ci/libraries/_initialization.sh

```diff
@@ -223,94 +275,190 @@ function initialize_common_environment {
     export CI_BUILD_ID="0"
 }
 
+function initialize_git_variables() {
+    # SHA of the commit for the current sources
+    COMMIT_SHA="$(git rev-parse HEAD || echo "Unknown")"
+    export COMMIT_SHA
+}
+
+function initialize_github_variables() {
+    # Defaults for interacting with GitHub
+    export GITHUB_REPOSITORY=${GITHUB_REPOSITORY:="apache/airflow"}
+    export GITHUB_REGISTRY=${GITHUB_REGISTRY:="docker.pkg.github.com"}
+    export USE_GITHUB_REGISTRY=${USE_GITHUB_REGISTRY:="false"}
+    export GITHUB_REGISTRY_PULL_IMAGE_TAG=${GITHUB_REGISTRY_PULL_IMAGE_TAG:="latest"}
+    export GITHUB_REGISTRY_PUSH_IMAGE_TAG=${GITHUB_REGISTRY_PUSH_IMAGE_TAG:="latest"}
+
+}
+
+# Common environment that is initialized by both Breeze and CI scripts
+function initialize_common_environment() {
```

Comment:
Thanks for breaking it down into smaller functions. It is now much clearer.

Reply:
Yeah. And unit testable. Soon.